Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -352,6 +356,10 @@ WarpX::computePhi (const amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho,
     const std::optional<amrex::Vector<amrex::FArrayBoxFactory const *> > eb_farray_box_factory;
 #endif
 
+    bool const is_solver_multigrid =
+        WarpX::poisson_solver_id == PoissonSolverAlgo::IntegratedGreenFunction ?
+        false : true;
```

Comment:
Clang tidy suggestion:
```suggestion
    bool const is_solver_multigrid =
        WarpX::poisson_solver_id != PoissonSolverAlgo::IntegratedGreenFunction;
```